src/CaptureClient/CaptureEventProcessor.cpp

```diff
@@ -81,6 +81,8 @@ class CaptureEventProcessorForListener : public CaptureEventProcessor {
       const orbit_grpc_protos::ErrorsWithPerfEventOpenEvent& errors_with_perf_event_open_event);
   void ProcessErrorEnablingOrbitApiEvent(
       const orbit_grpc_protos::ErrorEnablingOrbitApiEvent& error_enabling_orbit_api_event);
+  void ProcessErrorEnablingUserSpaceInstrumentationEvent(
+      const orbit_grpc_protos::ErrorEnablingUserSpaceInstrumentationEvent& error_event);
```

Comment:
Looking at the other parameter names, I see you didn't want to write `error_enabling_user_space_instrumentation_event`. Probably fair.

Reply:
Yep, felt a bit like repeating the type. Don't have a strong opinion. Will keep as is.